src/pages/lessons/[slug].tsx

```diff
@@ -221,13 +222,40 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     )
   }, [currentPlayerState])
 
-  const checkAutoPlay = () => {
+  const checkAutoPlay = async () => {
     if (autoplay && nextLesson) {
       console.debug('autoplaying next lesson', {nextLesson})
       track('autoplaying next video', {
         video: nextLesson.slug,
       })
-      router.push(nextLesson.path)
+
+      if (isFullscreen) {
```

Comment:
if we are fullscreenin we want to just move to the next video and make sure the player instance doesn't curn